Pyrado/pyrado/utils/experiments.py

```diff
@@ -202,8 +360,6 @@ def cpp_export(
                               This requires the experiment's XML configuration to be exported beforehand.
     :param policy_node_name: name of the control policies node in the XML file, e.g. 'policy' or 'preStrikePolicy'
     """
-    from pyrado.environments.rcspysim.base import RcsSim
```

Comment:
I do not like inline imports but as someone who does not use the RcsSim (nor have it installed) putting this line in the imports breaks simulacra.

Reply:
Merging error…